app/src/main/java/com/github/studenttimetracker/utils/CalendarUtils.java

```diff
@@ -12,15 +12,40 @@
 
 public class CalendarUtils {
 
-    public static final String timestampFormat = "dd/MM/yyyy HH:mm:ss";
+    public static final String timestampFormat = "yyyy-MM-dd HH:mm:ss";
     public static final String hourFormat = "HH:mm:ss";
 
+    public static String formatSeconds(int seconds) {
+        String result = "";
+
+        if (seconds <= 0){
+            return "0s";
+        }
+
+        int secondResult = seconds % 60;
+        if (secondResult > 0){
+            result = secondResult + "s" + result;
+        }
+
+        int minutes = seconds/60;
+        int minuteResult = minutes%60;
+        if (minuteResult > 0){
+            result = minuteResult + "m" + result;
+        }
+
+        int hours = minutes/60;
+        if (hours > 0){
+            result = hours + "h" + result;
+        }
+        return result;
+    }
+
     public static List<StatisticsQueryObject> getStatisticsQueryList(StatisticsPeriodType unitType, LocalDate startDate, LocalDate endDate) {
 
         List<StatisticsQueryObject> list = new ArrayList<>();
         switch (unitType) {
             case DAY:
-                while (endDate.isAfter(startDate) || endDate.isEqual(startDate)) {
+                while (endDate.isAfter(startDate.minusDays(1))) {
```

Comment:
to minusDays(1) jest po to aby przy zaznaczeniu startDate=endDate wychwycić przynajmniej jeden dzień, tak?

Reply:
akurat to było przy miesiącach - że jak jest startDate = 25 maja, a chcemy całe miesiące, to jeśli zaczniemy się cofać od dzisiaj (9 czerwca), to nie uchwyciłoby już maja. 
Identycznie dla roku.

Dla dni w sumie mogło zostać to z || endDate.isEqual(), ale dla spójności dodałem wszędzie